tests/acceptance/features/webUIPrivateLinks/accessingPrivateLinks.feature

```diff
@@ -18,4 +18,22 @@ Feature: Access private link
     Given the user navigates to the private link created by user "user1" for file "lorem.txt"
     When user "user1" has logged in using the webUI
     Then the app-sidebar for file "lorem.txt" should be visible on the webUI
-    And the "versions" details panel should be visible
\ No newline at end of file
+    And the "versions" details panel should be visible
+
+  Scenario: Access private link as a collaborator
+    Given user "user2" has been created with default attributes
+    And user "user1" has shared file "lorem.txt" with user "user2" with "read" permissions
+    And the user navigates to the private link created by user "user1" for file "lorem.txt"
```

Comment:
`And the user has navigated to the private link created by user "user1" for file "lorem.txt"`